internal/db/seeds/dao_pubkey.sql

```diff
@@ -0,0 +1,10 @@
+--
+-- Testing seed data, to execute this file during migration set DB_SEED variable to "dao_pubkey".
+--
+BEGIN;
+
+INSERT INTO pubkeys(id, account_id, name, body)
+VALUES (1, 1, 'firstkey', 'sha-rsa body')
```

Comment:
I'd try to use a valid public key here.
I'm not entirely against the idea of pubkey in the fixtures, but it is a bit of a stretch already (as not all the tests will need to have it in the DB)